### PR TITLE
Feature/chat backend

### PR DESCRIPTION
**Pull request by Sathvik-Chowdary-Veerapaneni**
- Sockets Implementation
- The chat session had basic interface to allow users communicate
- (User's) Open multiple chat windows (with same address link)

### CODE REVIEW

File: flask-discord/flaskd/templates/chat.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <title>Money Hunters Chat Application</title>
+  </head>
+  <body>
+
+    <h3 style='color: #ccc;font-size: 30px;'>No message's yet..</h3>
```
Comment (Nithish4603) [inline flask-discord/flaskd/templates/chat.html:8]: Great code! this is simple and effective

File: flask-discord/backend_tests/test_db.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+def test_get_email(app):
+    with app.app_context():
+        db = get_db()
+        email = "test@gmail.com"
+        user = db.execute("SELECT * FROM user WHERE email = ?", (email,)
+                ).fetchone()
+        assert len(user) != 0
+
+def test_get_uknown_user(app):
+    """ Tries to get unkown user from database. Should return an empty list """
```
Comment (Nithish4603) [inline flask-discord/backend_tests/test_db.py:16]: good unit testing.

**Review by Nithish4603 — APPROVED** [review]
Nice code works great and simple for a small group project.

**Review by spyralspy01 — APPROVED** [review]
Great implementation of code. this is a very good code for a small group. simple works as intended.

**Pull request merged**